### PR TITLE
[IMP] website_forum: improve demo data

### PR DESCRIPTION
**Pull request by niyasraphy**
it is specified as v8, generalized the demo data content to use odoo, instead of odoo v8

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rdeodoo — CHANGES_REQUESTED** [review]
Thanks for the proposal :+1: 

While doing some cleaning, you could remove the whitespace error a few lines below:
https://github.com/odoo/odoo/blame/b34c458311b4f9d73013b5aae67e61a3f15d1fe0/addons/website_forum/data/forum_demo.xml#L82
image

And also remove 2 of the 3 blank line on top of the file
https://github.com/odoo/odoo/blame/b34c458311b4f9d73013b5aae67e61a3f15d1fe0/addons/website_forum/data/forum_demo.xml#L4-L6

**Comment by niyasraphy** [comment]
@rdeodoo updated the pr as per suggested

File: addons/website_forum/data/forum_demo.xml [hunk 1]
```diff
@@ -79,7 +77,6 @@ The CMS editor in Odoo web is nice but I prefer Drupal for customization and the
             <field name="user_id" ref="base.user_demo"/>
             <field name="vote">1</field>
         </record>
-
```
Comment (rdeodoo) [inline addons/website_forum/data/forum_demo.xml:82]: I didn't meant to remove the line but to remove the multiple whitespace in it :)
Keep the blank line, it's how this file is defined: blank line + comment + record(s)
Then should be good :+1:

Comment (niyasraphy) [reply addons/website_forum/data/forum_demo.xml:82]: @rdeodoo sorry to understand it wrongly, updated again

**Review by rdeodoo — APPROVED** [review]
@robodoo r+ (probably the message could be improve one day to output a real clickable link, + maybe add the link of /app/ecommerce too on top of the website app one) but it's just some demo data, getting rid of the deprecated version is already nice
Thanks for the linting fixes

**Pull request closed without merging**